ui-lib/src/main/AndroidManifest.xml

```diff
@@ -14,10 +14,11 @@
 
         <activity
             android:name=".MainActivity"
+            android:configChanges="orientation|locale|layoutDirection|screenLayout|uiMode|colorMode|keyboard|screenSize"
```

Comment:
Please add an explanation comment above this change, ideally with a link to Compose documentation about why using such a configuration is a good approach here. Thank you.

Reply:
done ✅